src/testing/code-debugging.md

```diff
@@ -802,7 +832,12 @@ You can also debug a layout problem visually, by setting
 This is a boolean from the `rendering` library. It can be
 enabled at any time and affects all painting while it is true.
 The easiest way to set it is at the top of your `void main()`
-entry point. See an example in the following code:
+entry point.
+
+#### Example 2
```

Comment:
Calling this "Example 2" was a bit confusing to me. Example 2 of what? It doesn't seem to be related to "Example 1".